doc/page-features.h

```diff
@@ -2,6 +2,52 @@
 
 \page Features
 
+
+\section async Asynchronous methods
+
+In addition to standard methods, which block the caller until the method execution is finished and provide an immediate output value, Facelift
+supports asynchronous methods. Asynchronous methods have been introduced to support operations which potentially last a long time, or which are
+not able to produce an output value immediately.
+
+\subsection async-sub1 Declaration
+
+Methods can be marked as asynchronous using the "async" decorator, as in the following QFace definition:
+
+\code
+module facelift.example 1.0;
+
+interface MyInterface {
+
+    @async : true
+    int myAsyncMethod(int inputArgument);
+
+}
+\endcode
+
+\subsection async-sub2 Usage from QML UI code
+In order for the method's return value to be provided asynchronously, the method needs to be called with a javascript function object as argument, in addition to the
```

Comment:
Could you limit the line lengths a bit (I even used only 100 chars), so it can be read on laptop, as well?
s/javascript/JavaScript?